src/cache.rs

```diff
@@ -682,10 +682,11 @@ impl Cache {
                     let parse_errs = parse_errs.clone();
```

Comment:
It think the documentation of `resolve_imports` deserve to be updated, as you changed the return value, it's not in sync anymore.